bin/commands/deposit.js

```diff
@@ -502,8 +502,10 @@ async function redeemDeposit(tbtc, deposit, redemptionInfo) {
       }
       redemption.autoSubmit()
 
-      redemption.onWithdrawn(transactionID => {
-        resolve(standardDepositOutput(tbtc, deposit) + "\t" + transactionID)
+      redemption.onWithdrawn(async transactionID => {
+        resolve(
+          (await standardDepositOutput(tbtc, deposit)) + "\t" + transactionID
```

Comment:
Random note because I'm here---there's no need to `await` when passing a promise to `resolve`, as resolving with a promise will wait to resolve the outer promise until the inner promise is resolved (i.e., it flattens it out). See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve on resolving with `then`-ables.

Reply:
Thanks for sharing that! Would it resolve correctly also here where we want to add a string to the result of `await standardDepositOutput(tbtc, deposit)`?